### ISSUE TITLE
Usability fix: Optionally update linked template when transaction is changed

### ISSUE THREAD

**Issue by mtotschnig** [mtotschnig/MyExpenses#427]

**Comment by mtotschnig** [comment]
Another variant of this: When a transaction is deleted, we could offer to delete all instances of the same plan and the plan.

**Comment by nottheend** [comment]
This would be really nice.
Some recurring transactions may have exceptions or are dropped for one interval. Would be nice if MyExpenses could handle changes in calendar or update the calendar events

**Comment by mtotschnig** [comment]
@nottheend Sorry for reacting that late to this. MyExpenses already has the capability to handle exceptions. When you open the calendar view of a plan and long tap on a date, the context menu has a command for canceling the event for that date.

**Comment by nottheend** [comment]
You are right, thanks! Then no need to take previous comment into consideration
